apps/experiments/migrations/0077_alter_participant_external_chat_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.7 on 2024-04-11 10:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiments', '0076_create_participants'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='participant',
+            name='external_chat_id',
+            field=models.CharField(),
+        ),
+    ]
```

Comment:
this can be merged into 0075

Reply:
Ah, yes.

Reply:
When I put the schema + data changes together, I get
```
django.db.utils.OperationalError: cannot ALTER TABLE "experiments_participant" because it has pending trigger events
```

when it tries to alter the `external_chat_id` field on `Participant` to be non-nullable. According to a few places I saw (including [this one](https://stackoverflow.com/questions/12838111/django-db-migrations-cannot-alter-table-because-it-has-pending-trigger-events)). I suspect this is because everything is run in a transaction, so somehow it's not detecting the changes to the `external_chat_id` field, so it flags this as being null when we try to set it to not-null.

Keeping the schema and data migration out of each other's hair works just fine, so I'll do that.